### DISCUSSION TITLE
Is there a way to use a config.yaml file using the installer ?

### DISCUSSION THREAD

**Discussion by Vultik** [inventree/InvenTree#4625] [Q&A]
Is there a more convienent way than env var to customize the installation when using the installer setup ?
Can i use a filled config.yaml file before starting the instalation.
For example to provide DB config and admin credentials.

**Comment by matmair** [comment]
No. It is meant to just be a no-click, simple installer.
You can change the config file after the installer is finished the same as with any other install method.

**Reply by Vultik** [reply]
Yeah but by default it use a sqlite file for the DB.
I have seen elsewhere in the repo, an idea to create an interactive installer cli.
Could be great for having a ready to use instance after the setup

**Reply by matmair** [reply]
You are welcome to implement it. Creating an interactive menu system for multiple distros is not an easy feat.

**Reply by matmair** [reply]
You get a ready to use instance after the installer is run through

**Reply by Vultik** [reply]
In fact, it seems that you can provide a config.yaml file before installation : 
![image](https://user-images.githubusercontent.com/55872753/232844493-395fe90e-5558-489e-b5a5-57326149fb16.png)

**Reply by matmair** [reply]
you can also install ios apps on Android phones. I wrote the installer, it is not advised to pass random config files to the installer. This method is used to update existing installs. There are no checks to make sure you did not pass a broken config file.

**Reply by Vultik** [reply]
Ok i see
